.github/workflows/test-all-metadata.yml

```diff
@@ -67,6 +67,8 @@ jobs:
           sudo systemctl daemon-reload
           sudo systemctl restart docker
       - name: "🧪 Run '${{ matrix.coordinates }}' tests"
+        env:
+          GRAALVM_QUICK_BUILD: true
```

Comment:
Instead of using the env var, can we please set `quickBuild = true` in the NBT config?